src/hooks/useFeeTierDistribution.ts

```diff
@@ -112,9 +112,7 @@ function usePoolTVL(token0: Token | undefined, token1: Token | undefined) {
     }
 
     if (latestBlock - (_meta?.block?.number ?? 0) > MAX_DATA_BLOCK_AGE) {
-      ReactGA.exception({
-        description: `Graph stale (latest block: ${latestBlock})`,
-      })
+      ReactGA.event('exception', { description: `Graph stale (latest block: ${latestBlock})` })
```

Comment:
Does this need a 'fatal' property?

Reply:
It's not required so I read that as default to false, but happy to change if you prefer https://developers.google.com/analytics/devguides/collection/ga4/exceptions#exception_parameters